scripts/tasks/eslint.ts

```diff
@@ -4,9 +4,25 @@ import * as constants from './eslint-constants';
 import * as fs from 'fs';
 
 const files = [path.join(process.cwd(), constants.directory)];
+// used specifically for v9 packages
 const storiesPath = path.join(process.cwd(), 'stories');
+const hasStorySubfolder =
+  fs.existsSync(storiesPath) &&
+  fs
+    .readdirSync(storiesPath)
+    .map(subfolder => {
+      const storyPath = path.join(storiesPath, subfolder);
+      const storyFiles = !path.extname(storyPath).length ? fs.readdirSync(storyPath) : [];
 
-if (fs.existsSync(storiesPath)) {
+      if (storyFiles.length) {
+        return storyFiles.map(fileName => path.join(storiesPath, fileName));
+      }
+      return storyPath;
+    })
+    .flat()
+    .some(fileName => constants.extensions.includes(path.extname(fileName)));
```

Comment:
Added this extra check which makes sure a valid file exists within the `stories/` subfolder because linter errors out when running for this package after the migration since the `stories/` subfolder won't contain any of the valid extensions (.ts,.tsx,.js,.jsx) and only contains one story with a `.mdx` extension. `just-scripts` `eslintTask` doesn't seem to support the `no-error-on-unmatched-pattern` eslint CLI argument so this is a workaround.

![image](https://user-images.githubusercontent.com/8649804/204640087-8ab5fd28-44bb-49e0-a39b-8fb93452c57b.png)

Reply:
This approach causes quite the pipeline job slowdown 😰 so will need to rethink

Reply:
Decided to move the problematic `FocusIndicator.stories.mdx` from `@fluentui/react-tabster` to `@fluentui/react-components` package as part of the [`Accessibility` ](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-components/src/Concepts/Accessibility) and remove the storybook setup from `@fluentui/react-tabster` instead.